### ISSUE TITLE
I/O error html/strikethrough

### ISSUE THREAD

**Issue by steve-kasica** [romanticcircles/Stylesheets#3]
Looks like it's breaking on `<del rendition="strikethrough">`.

Error message from Oxygen
```
System ID: /Users/steve/Dev/romantic-circles/Stylesheets/html/html_header.xsl
Scenario: Romantic Cricles, to HTML
XML file: /Users/steve/Dev/romantic-circles/rc-tei/editions/Southey_Letters/Part_2/XML/letterEEd.26.346.xml
XSL file: /Users/steve/Dev/romantic-circles/Stylesheets/html/html.xsl
Engine name: Saxon-PE 12.3
Severity: fatal
Problem ID: FODC0002
Description: I/O error reported by XML parser processing file:/Users/steve/Dev/romantic-circles/Stylesheets/html/strikethrough: /Users/steve/Dev/romantic-circles/Stylesheets/html/strikethrough (No such file or directory)
Start location: 101:51
URL: http://www.w3.org/TR/2005/WD-xpath-functions-20050211/#ERRFODC0002
```

**Comment by steve-kasica** [comment]
Likewise on letterEed.26.541.xml

```
System ID: /Users/steve/Dev/romantic-circles/Stylesheets/html/html_header.xsl
Scenario: Romantic Cricles, to HTML
XML file: /Users/steve/Dev/romantic-circles/rc-tei/editions/Southey_Letters/Part_2/XML/letterEEd.26.541.xml
XSL file: /Users/steve/Dev/romantic-circles/Stylesheets/html/html.xsl
Engine name: Saxon-PE 12.3
Severity: fatal
Problem ID: FODC0002
Description: I/O error reported by XML parser processing file:/Users/steve/Dev/romantic-circles/Stylesheets/html/strikethrough: /Users/steve/Dev/romantic-circles/Stylesheets/html/strikethrough (No such file or directory)
Start location: 101:51
URL: http://www.w3.org/TR/2005/WD-xpath-functions-20050211/#ERRFODC0002
```

**Comment by steve-kasica** [comment]
Resolved See #1
